CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+### 1.1.0
+Add `folsom-semantic-metrics` module containing a [semantic-metrics][]-based implementation of Folsom's `Metrics`.
```

Comment:
I don't think we need a new release for this

Reply:
I can change this, but do you think it’s still useful to mention this in the changelog somewhere?